src/reps/grip-array.js

```diff
@@ -39,14 +39,12 @@ let GripArray = React.createClass({
   },
 
   getTitle: function (object, context) {
-    let objectLink = this.props.objectLink || span;
-    if (this.props.mode !== MODE.TINY) {
-      let title = this.props.title || object.class || "Array";
-      return objectLink({
-        object: object
-      }, title, " ");
+    if (this.props.mode === MODE.TINY) {
+      return "";
     }
-    return "";
+
+    let title = `${this.props.title || object.class || "Array"} `;
+    return this.safeObjectLink({}, title);
```

Comment:
Ah good point! In this case maybe still concatenate  the blank space with the title in a second step.